EntAppProject/JBIntroSurvey.html

```diff
@@ -1,39 +1,29 @@
-<!DOCTYPE html>
-
-<html>
-
-<head>
-
-<title>JobBook - Introduction Survey</title>
-
-<link rel="stylesheet" href="JobBookCSS.css">
-
-</head>
-
-<body>
- 
-
-<h1><span>Job</span>Book</h1>
-<h2>Welcome to JobBook</h2>
- 
-<p>
-In order to best assist you with finding jobs, please check the boxes that best

```

Comment:
This is just a reminder that although you have the content of the `<p>` tag broken up into multiple lines, the browser will read this content as one straight block of text with no manual breaking points. If that is not what you want, you will need to add some styles to the `<p>` tag to make it utilize text-wrapping.